services/common/src/redux/actionCreators/userActionCreator.js

```diff
@@ -1,20 +0,0 @@
-import { ENVIRONMENT } from "@mds/common/constants/environment";
-import { request, success, error } from "../actions/genericActions";
-import { NetworkReducerTypes } from "@mds/common/constants/networkReducerTypes";
-import * as userActions from "../actions/userActions";
-import * as API from "@mds/common/constants/API";
-import { createRequestHeader } from "../utils/RequestHeaders";
-import CustomAxios from "../customAxios";
-
-// This file is anticipated to have multiple exports
-// eslint-disable-next-line import/prefer-default-export
-export const fetchCoreUsers = () => (dispatch) => {
-  dispatch(request(NetworkReducerTypes.GET_CORE_USERS));
-  return CustomAxios()
-    .get(ENVIRONMENT.apiUrl + API.CORE_USER, createRequestHeader())
-    .then((response) => {
-      dispatch(success(NetworkReducerTypes.GET_CORE_USERS));
```

Comment:
I am delighted to delete these files, buuuut can we also then delete, say, NetworkReducerTypes.GET_CORE_USERS and any other unused references?

Reply:
Yeah...I started going down a delete path and got nervous about being overzealous.  Probably though.